Sources/DolbyIORTSCore/StreamCoordinator.swift

```diff
@@ -54,49 +50,83 @@ open class StreamCoordinator {
         self.subscriptionManager.delegate = self
 
         startStateObservation()
+        startStateMachineTasksSerialExecutor()
     }
-    #endif
 
     private func startStateObservation() {
         Task {
             await stateMachine.statePublisher
                 .sink { [weak self] state in
                     guard let self = self else { return }
+                    
+                    switch state {
+                    case let .error(state):
```

Comment:
This `state` is easy to be confused with the `state` above. Rename it.

Reply:
Done, renamed it to `errorState`